analytics-integrations/localytics/src/main/java/com/segment/analytics/internal/integrations/LocalyticsIntegration.java

```diff
@@ -128,7 +128,14 @@
     super.track(track);
     Properties props = track.properties();
     setContext(track.context());
-    Localytics.tagEvent(track.event(), props.toStringMap());
+
+    final long revenue = (long) (props.revenue() * 100);
```

Comment:
Good point, I think if you pass it in as under a cent, it's reasonable for us to just round to 0 cents. Hardly an issue in real code I think though, so I'm fine leaving it as is.